system/handlers/admin/DataManager.cfc

```diff
@@ -1035,6 +1035,53 @@ component extends="preside.system.base.AdminHandler" {
 		return renderView( view="admin/datamanager/versionNavigator", args=args );
 	}
 
+	private string function groupVersionNavigator( event, rc, prc, args={} ) {
+		var selectedVersion = Val( args.version ?: "" );
+		var objectName      = args.object ?: "";
+		var id              = args.id     ?: "";
+		var setting         = structKeyList( event.getCollectionForForm( "system-config.#args.id#" ) );
+		var maxVersion      = presideObjectService.getNextVersionNumber();
```

Comment:
This should be avoided. This will grow the version table unnecessarily. Instead of this (and the query of queries below), we should add a method to the sysConfig service that calculates the max version for a category. This should be achievable with a single regular selectData query I should think. Rough pseudocode:

```
var version = $getPresideObject( versionObjectNameForSysConfgTable ).selectData(
      selectFields = [ "Max( _version_number ) as max_version_number" ]
    , filter = { category = arguments.category }
);

return Val( version.max_version_number );
```